forklift/CloudLocatorsPallet.py

```diff
@@ -122,13 +113,11 @@ def process(self) -> None:
             #: copy rebuilt locator back
             self.copy_locator_to(rebuild_path, locator, path_to_locators)
 
-            for locator_part in path_to_locators.glob(f"{locator}*"):
-                self.log.debug("uploading %s", locator_part)
-
-                blob = self.bucket.blob(locator_part.name)
-                blob.upload_from_filename(locator_part)
+            #: upload them to all cloud projects
+            self._upload_locators_to_cloud_storage(list(path_to_locators.glob(f"{locator}*")))
 
-            self.publisher.publish(self.topic, data=bytes(), locator=locator)
+            self.cloud_services_prod.publisher.publish(self.cloud_services_prod.topic, data=bytes(), locator=locator)
+            self.cloud_services_dev.publisher.publish(self.cloud_services_dev.topic, data=bytes(), locator=locator)
```

Comment:
Do you need to add a try/catch around this in case the upload fails?

Reply:
i suppose if the prod upload fails the pallet will fail so i don't think we'd want to catch anything. If dev failed to upload, which we silently caught, it will upload and deploy the existing bucket locators which I think is harmless.